### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ many users will want iris and/or xarray as well:
 
 
 ```
-conda install -c conda-forge -c ioam holoviews>=1.6.2 geoviews
```
Comment (jbednar) [inline README.md]: Shouldn't it still enforce *some* minimal compatiable version of hv?  Or are you counting on conda automatically suggesting updates anyway?

Comment (philippjfr) [inline README.md]: Right, if you list a package you get the latest version.

File: doc/Gridded_Datasets_I.ipynb [hunk 2]
```diff
@@ -318,11 +318,8 @@
...
     "%%opts Curve [aspect=2 xticks=4 xrotation=15] Points (color='k')\n",
     "temp_curve = hv.Curve(xr_dataset.select(longitude=0, latitude=10), kdims=['time'])\n",
-    "\n",
-    "temp_maps = [cb.to(gv.Image,['longitude', 'latitude']) * gv.Points([(0,10)], crs=crs.PlateCarree()) \n",
-    "             for cb in [xr_dataset, air_temperature]]\n",
-    "\n",
-    "hv.Layout(temp_maps + [temp_curve]).cols(2)"
+    "temp_map = xr_dataset.to(gv.Image,['longitude', 'latitude']) * gv.Points([(0,10)], crs=crs.PlateCarree())\n",
+    "temp_map + temp_curve"
    ]
```
Comment (jbednar) [inline doc/Gridded_Datasets_I.ipynb]: The new version looks much better, though  I can't see why the convoluted one in the old version was ever needed!

File: geoviews/plotting/bokeh/__init__.py [hunk 3]
```diff
@@ -37,7 +37,8 @@ class GeoPlot(ElementPlot):
                                         BoxZoomTool(match_aspect=True), 'reset'],
         doc="A list of plugin tools to use on the plot.")
 
-    show_grid = param.Boolean(default=False)
+    show_grid = param.Boolean(default=True, doc="""
+        Whether to show a gridlines on the plot.""")
```
Comment (jbednar) [inline geoviews/plotting/bokeh/__init__.py]: Whether to show gridlines...

(Direct push ok.)

**Comment by jbednar** [comment]
All looks good, thanks!

File: doc/Gridded_Datasets_I.ipynb (hunk 2 above)
Comment (philippjfr) [inline doc/Gridded_Datasets_I.ipynb]: It simplified the old example, it was a bit clunky.

File: README.md (hunk 1 above)
Comment (jbednar) [inline README.md]: Well, you get the latest version unless that conflicts with some other package's requirement, which I guess we're counting on not happening in this case.

**Pull request merged**